docs/api/target.rst

```diff
@@ -355,3 +355,62 @@ Internally created variables (NAME = "foo"):
 
 Internal variable names are prefixed with ``_`` to avoid collision with input parameters.
 
+.. _blt_import_library:
+
+blt_import_library
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_import_library( NAME                     <libname>
+                        DEPENDS_ON               [dep1 [dep2 ...]]
+                        INCLUDES                 [include1 [include2 ...]]
+                        TREAT_INCLUDES_AS_SYSTEM [ON|OFF]
+                        FORTRAN_MODULES          [path1 [path2 ..]]
+                        LIBRARIES                [lib1 [lib2 ...]]
+                        COMPILE_FLAGS            [flag1 [flag2 ..]]
+                        LINK_FLAGS               [flag1 [flag2 ..]]
+                        DEFINES                  [def1 [def2 ...]] )
+
+Creates a CMake library target from a list of existing library files or targets.
+
+NAME
+  Name of the created CMake target
+
+DEPENDS_ON
+  List of CMake targets that this library depends on
+
+INCLUDES
+  List of include directories to be inherited by dependent targets
+
+TREAT_INCLUDES_AS_SYSTEM
+  Whether to inform the compiler to treat this library's include paths
+  as system headers
+
+FORTRAN_MODULES
+  FORTRAN module directories to be inherited by dependent targets
+
+LIBRARIES
+  List of CMake targets and library files (.a/.so/.lib/.dll) that make up
+  this library
+
+COMPILE_FLAGS
+  List of compiler flags to be inherited by dependent targets
+
+LINK_FLAGS
+  List of linker flags to be inherited by dependent targets
+
+DEFINES
+  List of compiler defines to be inherited by dependent targets
+
+This behaves similarly to :ref:`blt_register_library`, but because it defines
```

Comment:
I think the following could benefit from a rewrite since it focuses more on implementation details than what it actually provides and when one would want to use it.

Since we want `blt_import_library` to be the new standard, we should not be defining it in terms of the old version.
Rather we should define what `blt_import_library` does and modify the `blt_register_library` writeup to indicate when one would need/prefer to use the latter.